website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -145,30 +146,17 @@ Once the API is defined with spec/status fields and CRD validation markers, the
 $ make manifests
 ```
 
-This makefile target will invoke controller-gen to generate the CRD manifests at `config/crd/bases/cache.example.com_memcacheds.yaml`.
-
-#### OpenAPI validation
-
-OpenAPIv3 schemas are added to CRD manifests in the `spec.validation` block when the manifests are generated. This validation block allows Kubernetes to validate the properties in a Memcached Custom Resource when it is created or updated.
-
-Markers (annotations) are available to configure validations for your API. These markers will always have a `+kubebuilder:validation` prefix.
+This makefile target will invoke [controller-gen][controller_tools] to generate the CRD manifests at `config/crd/bases/cache.example.com_memcacheds.yaml`.
 
-Usage of markers in API code is discussed in the kubebuilder [CRD generation][generating-crd] and [marker][markers] documentation. A full list of OpenAPIv3 validation markers can be found [here][crd-markers].
+**Note** If you would like to define validations for the CRD's fields see the [OpenAPI valiation][openapi-validation] doc.
```

Comment:
CRD validation is already "defined" by running `controller-gen`, so this should just be a note about reading further
```suggestion
**Note** If you would like to read further into CRD validation, see the [OpenAPI valiation][openapi-validation] doc.
```